src/QMCDrivers/QMCDriverNew.cpp

```diff
@@ -404,6 +404,11 @@ QMCDriverNew::AdjustedWalkerCounts QMCDriverNew::adjustGlobalWalkerCount(Communi
     app_warning() << "Walkers per rank (" << awc.walkers_per_rank[rank_id] << ") not divisible by number of crowds ("
                   << num_crowds << "). This will result in a loss of efficiency.\n";
 
+  if (awc.global_walkers % num_ranks || awc.walkers_per_rank[rank_id] % num_crowds)
```

Comment:
`awc.global_walkers % (num_ranks * num_crowds)` instead?

Reply:
I prefer to copy the exact logic used in the lines above for consistency. i.e. If either of those warnings are triggered then this new printout clearly will be. Makes for easier reading.

Reply:
I found the condition in the if statement wasn't aligned with the warning. I feel it should.